src/animation.cpp

```diff
@@ -0,0 +1,426 @@
+#include "game.h"
+#include "cata_tiles.h" // all animation functions will be pushed out to a cata_tiles function in some manner
```

Comment:
@narc0tiq Thanks for working that out while I was sleeping -- I'll put together a small PR to fix that for @tivec.